installscripts/jazz-terraform-unix-noinstances/chefProvisioner.tf

```diff
@@ -99,23 +101,41 @@ resource "null_resource" "configureJenkinsInstance" {
   }
 }
 
-resource "null_resource" "configureJenkinsDocker" {
-  count = "${var.dockerizedJenkins}"
-  depends_on = ["null_resource.preJenkinsConfiguration", "aws_elasticsearch_domain.elasticsearch_domain"]
-  // Chef Process inside docker
+data "external" "gitlabcontainer" {
+  count = "${var.scmgitlab}"
+  program = ["bash", "${var.configureGitlab_cmd}"]
+
+  query = {
+    passwd = "${var.cognito_pool_password}"
+    ip = "${aws_lb.alb_ecs_gitlab.dns_name}"
+    gitlab_admin = "${lookup(var.scmmap, "scm_username")}"
+  }
+  depends_on = ["null_resource.health_check_gitlab"]
+}
+
+resource "null_resource" "configureCodeqDocker" {
+  count = "${var.dockerizedSonarqube}"
   provisioner "local-exec" {
-    command = "bash ${var.launchJenkinsCE_cmd}"
+    command = "python ${var.configureCodeq_cmd} ${aws_lb.alb_ecs_codeq.dns_name} ${lookup(var.codeqmap, "sonar_passwd")}"
+  }
+  depends_on = ["null_resource.health_check_codeq"]
+}
+
+resource "null_resource" "configureCliJenkins" {
+  depends_on = ["null_resource.health_check_jenkins", "null_resource.preJenkinsConfiguration", "null_resource.configureJenkinsInstance"]
```

Comment:
Remove `null_resource.preJenkinsConfiguration` from `depends_on`, `configureJenkinsInstance` already depends on it so it's redundant here.

Reply:
Also, since `configureJenkinsInstance` is purely for the non-containerized version, can we give it a more obvious name like `configureExistingJenkinsServer`, and then call this step `configureJenkins` instead of `configureCliJenkins`?

Would help a ton with readability.

Reply:
NVM, just pushed this, small change.